### PR TITLE
[FIB]: Add more logging messages

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

**Comment by stcheng** [comment]
ECMP/LAG logging messages will be improved in a separate pull request.

File: ansible/roles/test/files/ptftests/fib_test.py [hunk 1]
```diff
@@ -239,19 +240,19 @@ def check_balancing(self, dest_port_list, port_hit_cnt):
...
-        total_hit_cnt = float(sum(port_hit_cnt.values()))
+        total_hit_cnt = sum(port_hit_cnt.values())
         for ecmp_entry in dest_port_list:
-            total_entry_hit_cnt = 0.0
+            total_entry_hit_cnt = 0
             for member in ecmp_entry:
                 total_entry_hit_cnt += port_hit_cnt.get(member, 0)
-            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, total_hit_cnt/len(dest_port_list))
+            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, float(total_hit_cnt)/len(dest_port_list))
             logging.info("%-10s \t %10d \t %10d \t %10s"
```
Comment (lguohan) [inline ansible/roles/test/files/ptftests/fib_test.py]: can you make it clear, this is the ecmp group

File: ansible/roles/test/files/ptftests/fib_test.py [hunk 2]
```diff
@@ -239,19 +240,19 @@ def check_balancing(self, dest_port_list, port_hit_cnt):
...
                          % (str(ecmp_entry), total_hit_cnt/len(dest_port_list), total_entry_hit_cnt, str(round(p, 4)*100) + '%'))
             result &= r
-            if len(ecmp_entry) == 1:
+            if len(ecmp_entry) == 1 or not total_entry_hit_cnt:
                 continue
             for member in ecmp_entry:
-                (p, r) = self.check_within_expected_range(port_hit_cnt.get(member, 0), total_entry_hit_cnt/len(ecmp_entry))
+                (p, r) = self.check_within_expected_range(port_hit_cnt.get(member, 0), float(total_entry_hit_cnt)/len(ecmp_entry))
                 logging.info("%-10s \t %10d \t %10d \t %10s"
```
Comment (lguohan) [inline ansible/roles/test/files/ptftests/fib_test.py]: can you make it clear, this is the lag member.

File: ansible/roles/test/files/ptftests/fib_test.py [hunk 3]
```diff
@@ -239,19 +240,19 @@ def check_balancing(self, dest_port_list, port_hit_cnt):
...
+            total_entry_hit_cnt = 0
             for member in ecmp_entry:
                 total_entry_hit_cnt += port_hit_cnt.get(member, 0)
-            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, total_hit_cnt/len(dest_port_list))
+            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, float(total_hit_cnt)/len(dest_port_list))
             logging.info("%-10s \t %10d \t %10d \t %10s"
                          % (str(ecmp_entry), total_hit_cnt/len(dest_port_list), total_entry_hit_cnt, str(round(p, 4)*100) + '%'))
             result &= r
-            if len(ecmp_entry) == 1:
+            if len(ecmp_entry) == 1 or not total_entry_hit_cnt:
```
Comment (lguohan) [inline ansible/roles/test/files/ptftests/fib_test.py]: total_entry_hit_cnt is an integer, so it is better to use (total_entry_hit_cnt == 0). "not" is used for bool.

Comment (stcheng) [inline ansible/roles/test/files/ptftests/fib_test.py]: ok

File: ansible/roles/test/files/ptftests/fib_test.py (hunk 1 above)
Comment (stcheng) [inline ansible/roles/test/files/ptftests/fib_test.py]: updated

File: ansible/roles/test/files/ptftests/fib_test.py (hunk 2 above)
Comment (stcheng) [inline ansible/roles/test/files/ptftests/fib_test.py]: updated

**Pull request merged**